src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,126 +4,198 @@
+        .whileTrue(new RunCommand(() -> intake.manipulates(-1.0))).onFalse(armPositions.armStowCommand())
+        .whileFalse(new RunCommand(() -> intake.manipulates(direction)));
+    // Pick up Cube
+    new JoystickButton(m_driverController, XboxController.Button.kX.value).onTrue(armPositions.armPickUpCubeCommand())
+        .whileTrue(new RunCommand(() -> intake.manipulates(1.0))).onFalse(armPositions.armStowCommand())
+        .whileFalse(new RunCommand(() -> intake.manipulates(direction)));
+    // Shelf
+    new JoystickButton(m_operatorController, XboxController.Button.kLeftBumper.value)
+        .onTrue(armPositions.armPickUpFromShelf()).whileTrue(new RunCommand(() -> intake.manipulates(-1.0)));
+  }
 
-          //Make sure that all buttons are unique
+  private void configureOpperatorBindings() {
 
-    new JoystickButton(m_driverController, XboxController.Button.kLeftBumper.value).onTrue(new InstantCommand(() -> intake.toggleIntake(), intake));
+    new JoystickButton(m_operatorController, XboxController.Button.kY.value)
+        .onTrue(armPositions.armScoreConeHighCommand());
+    // cone mid
+    new JoystickButton(m_operatorController, XboxController.Button.kB.value)
+        .onTrue(armPositions.armScoreConeMidCommand());
+    // cube High
+    new JoystickButton(m_operatorController, XboxController.Button.kX.value)
+        .onTrue(armPositions.armScoreCubeHighCommand());
+    // cube mid
+    new JoystickButton(m_operatorController, XboxController.Button.kA.value)
+        .onTrue(armPositions.armScoreCubeMidCommand());
 
-new JoystickButton(m_driverController, XboxController.Button.kY.value).whileTrue(new InstantCommand(() -> m_arm.moveArmToPosition(armPosition.INTAKE_ARM_POSITION_GROUND), m_arm));
+  }
 
-new JoystickButton(m_driverController, XboxController.Button.kA.value).whileTrue(new InstantCommand(() -> m_arm.moveArmToPosition(armPosition.INTAKE_ARM_POSITION_STOWED), m_arm));
+  public double getSparkMax() {
+    return m_arm.getArmAbsoluteEncoder();
+  }
 
-    new JoystickButton(m_driverController, XboxController.Button.kB.value).whileTrue(new InstantCommand(() -> m_arm.moveArmToPosition(armPosition.SCORING_ARM_POSITION_MID), m_arm));
-  
+  public double getElbowSparkMax() {
+    return m_arm.getElbowAbsoluteEncoder();
+  }
+
+  public void getIdleMode(IdleMode idleMode) {
+    m_arm.getSparkStatus(idleMode);
   }
 
   /**
    * Use this to pass the autonomous command to the main {@link Robot} class.
    *
    * @return the command to run in autonomous
    */
+
   public Command getAutonomousCommand() {
-    return null;
+    return AutoChooser.getAuton();
+  }
+
+  public void putAccel(){
+  SmartDashboard.putNumber("Gyro Accel X", m_gyro.getAccelX());
+  SmartDashboard.putNumber("Gyro Accel y", m_gyro.getAccelY());
+  SmartDashboard.putNumber("Gyro Accel z", m_gyro.getAccelZ());
+  }
+
+  public void autonInit() {
+    m_robotDrive.calibrateGyro();
+    m_robotDrive.stop();
+  }
+
+  public void getPitch(){
```

Comment:
I would recommend renaming this function to putPitchOnDashboard because it doesn't return anything.